Vaishnavi/ass3.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Register</h1>
+        <form>
+            <label>First Name</label>
+            <input id="user" name="user first name" type="text" required />
+            <label>Last Name</label>
+            <input id="user" name="user last name" type="text" required />
+            <br/>
+            <label>Gender</label>
+            <input type="radio" name="gender" value="Male" />Male
+            <input type="radio" name="gender" value="Female"/>Female
+            <input type="radio" name="gender" value="others"/>others<br/>
+            <label>Email</label>
+            <input type="email" name="email"/>
+<label>Password</label>
+            <input type="password" name="password"/>

```

Comment:
Given condition is not satisfied, refer conditions.txt

Reply:
@vaishnavi2002 This is still pending.